docs/sources/flow/reference/components/prometheus.exporter.kafka.md

```diff
@@ -27,30 +27,41 @@ prometheus.exporter.kafka "LABEL" {
-| `allow_concurrency`         | `bool`          | If set to true, all scrapes trigger Kafka operations. Otherwise, they will share results. WARNING: Disable this on large clusters.                                                 | `true`  | no       |
-| `max_offsets`               | `int`           | The maximum number of offsets to store in the interpolation table for a partition.                                                                                                 | `1000`  | no       |
-| `prune_interval_seconds`    | `int`           | How frequently should the interpolation table be pruned, in seconds.                                                                                                               | `30`    | no       |
-| `topics_filter_regex`       | `string`        | Regex filter for topics to be monitored.                                                                                                                                           | `.*`    | no       |
-| `groups_filter_regex`       | `string`        | Regex filter for consumer groups to be monitored.                                                                                                                                  | `.*`    | no       |
+| Name                          | Type            | Description                                                                                                                                                                        | Default | Required |
+| ----------------------------- | --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- | -------- |
+| `kafka_uris`                  | `array(string)` | Address array (host:port) of Kafka server.                                                                                                                                         |         | yes      |
+| `instance`                    | `string`        | The`instance`label for metrics, default is the hostname:port of the first kafka_uris. You must manually provide the instance value if there is more than one string in kafka_uris. |         | no       |
+| `use_sasl`                    | `bool`          | Connect using SASL/PLAIN.                                                                                                                                                          |         | no       |
+| `use_sasl_handshake`          | `bool`          | Only set this to false if using a non-Kafka SASL proxy.                                                                                                                            | `true`  | no       |
```

Comment:
Is the change in default value intentional? Should it be documented in the changelog?

Reply:
This value should was actually true by default already, it was just wrongly documented (thanks for pointing it out, I did not even realised). https://github.com/grafana/agent/blob/main/internal/component/prometheus/exporter/kafka/kafka.go#L18
I think that because it's just a doc mistake we don't need to document it in the changelog